assets/app/view/game_card.rb

```diff
@@ -10,6 +10,7 @@ class GameCard < Snabberb::Component
     needs :user
     needs :gdata # can't conflict with game_data
     needs :confirm_delete, store: true, default: false
+    needs :confirm_kick, store: true, default: false
```

Comment:
let's make the types consistent, default should be nil

Reply:
Good point

Should I also change it for :confirm_delete?

Reply:
Yes.